frb_example/with_flutter/pubspec.yaml

```diff
@@ -49,6 +50,8 @@ dev_dependencies:
   # package. See that file for information about deactivating specific lint
   # rules and activating additional ones.
   flutter_lints: ^1.0.0
+  freezed: ^2.0.3+1
```

Comment:
Hmm why add freezed & build_runner to `with_flutter` example? Seems you have not changed anything in this example

Reply:
I thought there was something wrong with build_runner for this example. Should I remove? This is going to be needed if someone adds something that needs build_runner accidentally

Reply:
Yes, but it is not needed currently. Anyway you can keep it, no problem